src/commands/lambda/flare.ts

```diff
@@ -771,6 +784,75 @@ export const sleep = async (ms: number) => {
   await new Promise((resolve) => setTimeout(resolve, ms))
 }
 
+/**
+ * Get the framework used based on the files in the directory
+ * @returns the framework used or undefined if no framework is found
+ */
+export const getFramework = () => {
+  const files = new Set(fs.readdirSync(process.cwd()))
+
+  // Check for serverless framework
+  const serverlessFiles = ['serverless.yml', 'serverless.yaml']
+  for (const file of serverlessFiles) {
+    if (files.has(file)) {
+      return 'Serverless Framework'
```

Comment:
Turn into `Enum`

Reply:
Is this what you were looking for?
```
enum Frameworks {
  Serverless_Framework = 'Serverless Framework',
  AWS_CDK = 'AWS CDK',
  AWS_CloudFormation = 'AWS CloudFormation',
  Unknown = 'Unknown',
}
```

And then: `return Frameworks.Serverless_Framework` instead of returning a string literal.

Reply:
Camel case for Enums: https://www.typescriptlang.org/docs/handbook/enums.html